### PR TITLE
blog: update Understanding Ark Liquidity Requirements

### PR DESCRIPTION
**Pull request by bordalix**
PR for discussion.

@tiero what do you think?

### CODE REVIEW

File: website/blog/2023-12-14-liquidity-requirements/index.md [hunk 1]
```diff
@@ -29,7 +29,139 @@ It's recommended to read the [nomenclature](/learn/nomenclature) first.
...
+    end
+    forfeit(Forfeit to ASP)
+    change2 --> forfeit
+  end
+  pool --> VTXOs
+  pool2 --> VTXOs
+  pool3 --> new_VTXOs
+```
+
+Summary:
```
Comment (tiero) [inline website/blog/2023-12-14-liquidity-requirements/index.md]:
```suggestion
### Summary
```

File: website/blog/2023-12-14-liquidity-requirements/index.md [hunk 2]
```diff
@@ -29,7 +29,139 @@ It's recommended to read the [nomenclature](/learn/nomenclature) first.
...
+While onboarding does not require any liquidity from the ASP, next payments will.
+:::
+
+### The liquidity problem
+
+If the ASP exhausts its liquidity, it will have to cease accepting new payments and will be compelled to wait for the timelocks on the on-chain transactions to expire in order to regain liquidity.
+
+To avoid the ASP from running out of funds, it must be able to anticipate how much liquidity will be required throughout that period.
+
+As we observed, the liquidity necessary will rely on the capital onboarded and then some; so, let's suppose it will be the consequence of a function of capital onboarded and a percentage of this capital utilized during a round.
```
Comment (tiero) [inline website/blog/2023-12-14-liquidity-requirements/index.md]: > and then some

What do you mean here?

File: website/blog/2023-12-14-liquidity-requirements/index.md [hunk 3]
```diff
@@ -29,7 +29,139 @@ It's recommended to read the [nomenclature](/learn/nomenclature) first.
...
+
+Therefore, how much BTC can an ASP accept from new users without risking not having enough capital to finance transfers within Ark? In other words, what proportion of BTC in Ark is transferred in a 1-month period?
```
Comment (tiero) [inline website/blog/2023-12-14-liquidity-requirements/index.md]: > how much BTC can an ASP accept from new users

This is not clear, you mean "request of payments/transactions"?

File: website/blog/2023-12-14-liquidity-requirements/index.md (hunk 2 above)
Comment (bordalix) [reply website/blog/2023-12-14-liquidity-requirements/index.md]: I rephrase it.

**Pull request merged**